tests/helper_functions.py

```diff
@@ -44,8 +43,12 @@ def random_int(lo=1, hi=1e10):
     return random.randint(lo, hi)
 
 
-def random_str(size=10, chars=string.ascii_uppercase + string.digits):
-    return "".join(random.choice(chars) for _ in range(size))
+def random_str(size=10):
+    """UUID4 generated strings have a high potential for collision at small sizes.
+    10 is lower bounds for this test harness utility."""
+    generated_string = uuid.uuid4().hex[:size]
+    assert len(generated_string) >= 10
```

Comment:
```suggestion
    msg = (
        "UUID4 generated strings have a high potential for collision at small sizes. "
        "10 is lower bounds for this test harness utility."
    )
    assert size >= 10, msg
    generated_string = uuid.uuid4().hex[:size]
```

Can we add a message to `assert`?

Reply:
changed!